src/components/AppProvider/AppProvider.tsx

```diff
@@ -12,7 +12,13 @@ import {
   polarisAppProviderContextTypes,
 } from './types';
 
-export default class AppProvider extends React.Component<AppProviderProps> {
+// The script in the styleguide that generates the Props Explorer data expects
+// a component's props to be found in the Props interface. This silly workaround
```

Comment:
I could be wrong but I recall it working on types as well e.g. `type Props = AppProviderProps`

Reply:
I tried that first and it didn't work :(

Reply:
Too bad :/